products/apache-ant.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Apache Ant
+category: app
+tags: apache build-tool java-runtime
+iconSlug: apacheant
+permalink: /ant
+alternate_urls:
+-   /apache-ant
+changelogTemplate: https://ant.apache.org/antnews.html
+versionCommand: ant -version
+releaseDateColumn: true
+
+identifiers:
+-   cpe: cpe:/a:apache:ant
+-   cpe: cpe:2.3:a:apache:ant
+
+auto:
+  methods:
+  -   maven: org.apache.ant/ant
+
+releases:
+-   releaseCycle: "1.10"
+    releaseDate: 2016-12-31
+    eol: false
+    latest: "1.10.15"
+    latestReleaseDate: 2024-08-29
+
+-   releaseCycle: "1.9"
+    releaseDate: 2013-03-07
+    eol: 2024-06-19
+    latest: "1.9.16"
+    latestReleaseDate: 2021-07-13
+
+-   releaseCycle: "1.8"
+    releaseDate: 2010-02-08
+    eol: true
```

Comment:
Setting the EOL(1.8) to releaseDate(1.9) seems reasonable here, since a formal announcement wasn't made, and 1.9 was released a year after the last release on 1.8.

```suggestion
    eol: 2013-03-07
```